e2e/cypress/integration/enterprise/guest_accounts/guest_invitation_ui_spec.js

```diff
@@ -268,7 +270,7 @@ describe('Guest Account - Guest User Invitation Flow', () => {
 
             invitePeople(user.email, 1, user.username);
 
-            verifyInvitationSuccess(user.username, 'This guest has been added to the team and channel.', true);
+            verifyInvitationSuccess(user.username, 'An invitation email has been sent.');
```

Comment:
The new change here is failing for me and what was here formally (asserting "This guest has been added to the team and channel") is in fact what still passes.

Reply:
As per the testcase, we are adding a user who is not part of the team. Hence the message should be "The guest has been added to the team and channel". If you are seeing a different behavior, it might be because of a bug in the application. Can you please recheck and revert this.

Reply:
Updated